app/src/main/java/com/github/sdpcoachme/firebase/database/RealDatabase.kt

```diff
@@ -0,0 +1,8 @@
+package com.github.sdpcoachme.firebase.database
+
+class RealDatabase : Database {
+
+    override fun isExistingUser(email: String): Boolean {
+        return true
+    }
```

Comment:
Maybe we can add a TODO here if we need to implement this function later ?

Reply:
I agree with that